<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="consoleTextWidget" name="consoleWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>630</y>
      <width>1191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1191</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Прямое управление</string>
     </attribute>
     <widget class="directManageWidget" name="DMWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1131</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>G-код</string>
     </attribute>
     <widget class="viewWidget" name="vWidget" native="true">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>931</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="gCodeEditWidget" name="gCodeTextEdit" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="stopPushButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>520</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Стоп</string>
      </property>
     </widget>
     <widget class="QPushButton" name="runPushButton_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>520</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Пауза</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>520</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Диаметр пера</string>
      </property>
      <widget class="QDoubleSpinBox" name="penDSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Высота подъема по оси Z.&lt;/p&gt;&lt;p&gt;Определяет, на сколько поднимется &lt;/p&gt;&lt;p&gt;перо при переходе к следующей &lt;/p&gt;&lt;p&gt;линии.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>520</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Z подъема пера</string>
      </property>
      <widget class="QDoubleSpinBox" name="ZSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Высота подъема по оси Z.&lt;/p&gt;&lt;p&gt;Определяет, на сколько поднимется &lt;/p&gt;&lt;p&gt;перо при переходе к следующей &lt;/p&gt;&lt;p&gt;линии.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>520</y>
        <width>120</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Скорость рабочая</string>
      </property>
      <widget class="QDoubleSpinBox" name="FSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Скорость подачи.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>520</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Скорость перемещения</string>
      </property>
      <widget class="QDoubleSpinBox" name="FMSpinBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Скорость подачи.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="runPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Запуск</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Связь</string>
    </property>
    <addaction name="conntcnAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openAction"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="conntcnAction">
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>directManageWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/directmanagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>consoleTextWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>widgets/consoletextwidget.h</header>
  </customwidget>
  <customwidget>
   <class>viewWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/viewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>gCodeEditWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/gcodeeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
